README.md

```diff
@@ -1,5 +1,29 @@
-# Django ORM boilerplate for GitHub tasks
+# API Views
 
-- Warning: Use `pytest app` for testing - not simple `pytest`
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
\ No newline at end of file
+
+Now, you are going to implement views via class-based views.
+
+Create `Genre`, `Actor`, `CinemaHall` models and update `Movie` model to
+the ones you wrote in Django ORM module. Modules should have such fields:
+- `Actor`: `first_name`, `last_name`
+- `Genre`: `name`
+- `CinemaHall`: `name`, `rows`, `seats_in_row`
+- `Movie`: `title`, `description`, `actors`, `genres`, `duration`. (note: you 
+already have a new field here) 
+
+Create serializers for all these models.
```

Comment:
Please add here smth like this: *Do not use related serializers for ManyToMany Relations*